configs/xfce/CNCHI_UPDATER.pot

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
@Wyn10 This should be a relative symlink to the file that is in the main iso config directory. So after changing directories to `antergos-iso/configs/xfce` you would run this command to create the symlink: `ln -sr ../antergos/CNCHI_UPDATER.pot CNCHI_UPDATER.pot`